README.md

```diff
@@ -2,36 +2,29 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] Great work on implementing the before-execution time but I guess the in the second query the after-execution time is supposed to be less than 1000ms but in the screenshot, it is 1099ms 😨 Kindly make it less to follow the project requirements as suggested by the previous reviewer 👍

| Before | After |
| --- | --- |
| ![image](https://github.com/veronica365/database-vet-clinic/assets/93039359/e7abb8e5-e7ed-449e-a36d-30c72956b5b1) | ![image](https://github.com/veronica365/database-vet-clinic/assets/93039359/a3709bb9-3475-435f-b4eb-5a7d167f2bd3) |